utils/sanitiseMetadata.test.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable camelcase*/
+const { sanitiseMetadata } = require("./sanitiseMetadata");
+jest.mock("uuid/v1", () => () => 123);
+describe("sanitise Metadata", () => {
+  let defaultMetadata;
+
+  beforeEach(() => {
+    defaultMetadata = {
+      tx_id: 123,
+      jti: 123,
+      iat: 10000,
+      exp: 20000,
+      user_id: "UNKNOWN",
+      case_id: 123,
+      ru_ref: "12346789012A",
+      ru_name: "ESSENTIAL ENTERPRISE LTD",
+      trad_as: "ESSENTIAL ENTERPRISE LTD",
+      eq_id: 1,
+      collection_exercise_sid: 123,
+      period_id: "201605",
+      form_type: 1,
+      survey_url: `${process.env.PUBLISHER_URL}1`
```

Comment:
Do we need to set the env var for the test? Seems like we should be able to put whatever here?